README.md

```diff
@@ -78,6 +78,37 @@ Once connected, run the following make target:
   make local
 ```
 
+### Running Bulk Indexer
+
+#### Locally
+
+Build the bulk indexer by running the following command
+```
+  make reindex
+```
+Then run the executable  
+```
+  ./reindex
+```
+Please make sure your elasticsearch server is running locally on localhost:9092 and version of the server is 7.10, which is the current supported version.
```

Comment:
Should this be localhost:**9200** for Elasticsearch; I believe 9092 is kafka

Reply:
Done in update 1.